src/state/chromeStore.ts

```diff
@@ -0,0 +1,15 @@
+import { createStore } from 'jotai';
+import { activeModuleAtom, contextSwitcherOpenAtom } from './atoms';
+
+const chromeStore = createStore();
+
+// setup initial chrome store state
+chromeStore.set(contextSwitcherOpenAtom, false);
```

Comment:
@Hyperkid123 a few questions here:
1. In `src/state/atoms.ts` we already specify what appears to be a default value of `false`. What is the point of setting it again here? If this is required by jotai, can we somehow reduce the duplication?
2. Similar to my comments in `src/state/atoms.ts` we may want to consider how to organize these files to avoid merge conflicts in the future.

Reply:
> In src/state/atoms.ts we already specify what appears to be a default value of false. What is the point of setting it again here? If this is required by jotai, can we somehow reduce the duplication?

This is actually required. It has to be set.

But I think we might not even need the store TBH, we can work directly with the atoms. I have to rethink this a little bit.